web/server/routers/post.ts

```diff
@@ -353,14 +344,12 @@ export const postRouter = router({
         getsAlongWithYoungKids: {
           $in: getsAlongWith["getsAlongWithYoungKids"],
         },
-        houseTrained: { $in: houseTrained },
-        spayNeuterStatus: { $in: spayNeuterStatus },
       };
       let completeFilter;
```

Comment:
Sorry, I should have specified. You don't need to declare another variable here. Just work with `baseFilter`.